great_expectations/data_context/store/gx_cloud_store_backend.py

```diff
@@ -92,17 +89,13 @@ class GXCloudStoreBackend(StoreBackend, metaclass=ABCMeta):
 
     RESOURCE_PLURALITY_LOOKUP_DICT: bidict = bidict(  # type: ignore[misc] # Keywords must be str
         **{  # type: ignore[arg-type]
-            GXCloudRESTResource.BATCH: "batches",
             GXCloudRESTResource.CHECKPOINT: "checkpoints",
+            GXCloudRESTResource.DATASOURCE: "datasources",
             GXCloudRESTResource.DATA_ASSET: "data_assets",
             GXCloudRESTResource.DATA_CONTEXT_VARIABLES: "data_context_variables",
```

Comment:
Is `GXCloudRESTResource` and Enum? 
You might consider putting the lookup on the `GXCloudRESTResource` itself.

```python
>>> enum_value = GXCloudRestResource("datasource")
>>> enum_value.pluralize()
"datasources"
```

Reply:
ooh I like that but I don't know if we're consistently adding the same suffix here. I'll defer this to a later effort